gdx/src/com/badlogic/gdx/files/FileHandle.java

```diff
@@ -684,9 +685,7 @@ static public FileHandle tempFile (String prefix) {
 
 	static public FileHandle tempDirectory (String prefix) {
 		try {
-			File file = File.createTempFile(prefix, null);
-			if (!file.delete()) throw new IOException("Unable to delete temp file: " + file);
-			if (!file.mkdir()) throw new IOException("Unable to create temp directory: " + file);
+			File file = Files.createTempDirectory(prefix).toFile();
```

Comment:
This was added in Android API Level 26; I don't know if that is considered a viable level for the project currently. https://developer.android.com/reference/java/nio/file/Files#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute<?>...)

RoboVM doesn't appear to have code in it that provides java.nio.Files, but some of the compile-time code does use it (no idea if it works on an iOS device): https://github.com/MobiVM/robovm/search?q=createTempDirectory